Dockerfile

```diff
@@ -26,6 +26,9 @@ RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && \
     echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list && \
     apt-get update && apt-get install -y yarn
 
+# Install MuPdf for PDF previews
+RUN apt-get install mupdf mupdf-tools
```

Comment:
Is that the dev version? Also, you will need to update the [Dockerfile.production][1] to include it as well 

[1]: https://github.com/unboxed/bops/blob/master/Dockerfile.production

Reply:
Thanks for the reminder - will do that now!